packages/govuk-frontend/src/govuk/components/back-link/back-link.yaml

```diff
@@ -22,11 +22,9 @@ params:
 
```

Comment:
Could 'with custom link' also be hidden?

Reply:
Yeah, `with custom text and link` does nothing that can't be tested programatically.